packages/web-app-files/src/components/SideBar/TagsPanel.vue

```diff
@@ -88,6 +88,7 @@ export default defineComponent({
     const resource = computed<Resource>(() => unref(injectedResource))
     const selectedTags = ref<TagOption[]>([])
     const availableTags = ref<TagOption[]>([])
+    let allTags = []
```

Comment:
yes, will add type hinting. but it will not change, the values are set in the loadAvailableTagsTask function only, so I don't see the added value of a ref here.